cdm/core/src/main/java/thredds/inventory/CollectionManagerCatalog.java

```diff
@@ -146,6 +147,16 @@ public Object getAuxInfo() {
     public void setAuxInfo(Object info) {
       this.info = info;
     }
+
+    @Override
+    public void writeToStream(OutputStream outputStream) throws IOException {
+      throw new IOException("Writing MFileRemote not implemented. Filename: " + getName());
+    }
+
+    @Override
+    public void writeToStream(OutputStream outputStream, long offset, long maxBytes) throws IOException {
+      throw new IOException("Writing MFileRemote not implemented. Filename: " + getName());
+    }
   }
```

Comment:
I think this has the potential to break things down to line, but I also think that's a "down the line" problem that we can ignore for now.

Reply:
I could also implement these, it is probably not very hard. But not necessary for this bug fix so I could do it in a separate PR

Reply:
Yeah, up to you, but I think it's fine to wait and see how this first pass goes over in the TDS